spaces/domains.json

```
@@ -1,4 +1,5 @@
 {
+  "vote.ecc.capital": "empire-capital.eth"
```

A comma is missing at the end of the line of your sub-domain.
```suggestion
  "vote.ecc.capital": "empire-capital.eth",
```